tutorials/openstack_workload_sfc.md

```diff
@@ -0,0 +1,201 @@
+*Verify Binding*  
+
+
+## Validating Setup and Redirection  
+
+#### OpenStack Instances  
+
+After setting up OSC and deploying a **Distributed Appliance Instance**, verify the **Distributed Appliance Instance** was deployed on OpenStack.  
+* Log in to OpenStack and navigate to **Project** > **Compute** > **Instances**.  
+* Observe that the **Distributed Appliance Instance** is listed as an instance in addition to the `ATTACKER` and `VICTIM`. Ensure the **Distributed Appliance Instance** image name reflects previous naming actions taken such as the name of the **Distributed Appliance Instance** (myDA) and the name of the image uploaded in the **Service Function Catalog**.  
+
+![OpenStack Instances](images/os_vms_sfc.jpg)  
+*OpenStack Instances*  
+
+#### Validating Network Redirection
+
+In the case of using the dummy CirrOS image, when a policy is bound to a security group, no virtual machine can communicate to the protected virtual machine - the `VICTIM` - and any network traffic directed to the `VICTIM` will be intercepted by the **Distributed Appliance Instance** on the `INSPECTION NETWORK`. This is the expected behavior for the CirrOS image which implements a dummy behavior that always blocks traffic to the protected virtual machines. Let's take a look at the network traffic flow observed through the **Distributed Appliance Instance** when the security group is bound and when it is unbound.
+
+* On OSC, ensure that the security group is [bound](#user-content-8-bind-security-group).
+* In the **Overview** tab for the `VICTIM`, note the `INSPECTION NETWORK` IP address. The `ATTACKER` will attempt to send an HTTP request to this address.  
+* On OpenStack, navigate to **Project** > **Compute** > **Instances**, and then open the `ATTACKER` and **Distributed Appliance** instances in different browser tabs. Ensure the `VICTIM` is powered on.  
+* Switch to the **Console** view for both the `ATTACKER` and **Distributed Appliance** instances.  
+* From the `ATTACKER` console, send an HTTP request to the `VICTIM`. For example, use the command: `wget <VICTIM IP ADDRESS>`. Note that there should be no connection.  
+* From the **Distributed Appliance** console:  
+	* From the command line, enter `sudo ip link set eth1 up`.  
+	* Enter `sudo tcpdump -i eth1`.
+	* Observe the *redirected* packets coming from the `ATTACKER` IP address on the eth1 interface. This means the binding was a success.
+* Unbind the security group from OSC REST API for delete binding sfc with security group.  
+* Ensure you get response with status code 200.  
+
+![](images/unbind_sfc_sg.jpg)  
+*Unbind the Policy from the Security Group*  
+
+* From the `ATTACKER` console on OpenStack, send another HTTP request to the `VICTIM`.
+	* The HTTP request should be successful now that the security group is unbound.
+* From the **Distributed Appliance Instance**, there should be no redirected traffic from the `ATTACKER`.
+
+## Creating Chain of services.

```

Comment:
i think we can eliminate this section ad incorporate this info into the Define a Service Function Chain section.

Reply:
@arvindn05 Creation of multiple service function chain requires some steps like creation of DA, DS, importing another sensor image and binding. so for binding again the user need to refer step 9 , one has go back and forth for referring few steps so